data/mods/gen5/moves.js

```diff
@@ -934,6 +934,16 @@ let BattleMovedex = {
 		basePower: 40,
 		flags: {protect: 1, mirror: 1, sound: 1},
 	},
+	soak: {
+		inherit: true,
+		onHit(target) {
+			if (!target.setType('Water')) {
+				this.add('-fail', target);
```

Comment:
Why not `return false`?

Reply:
Oh, because the animation. I would suggest adding a comment to explain that.

Reply:
I've added some comments; I hope you like them.